docs/database-connection-string.md

```diff
@@ -1,85 +1,83 @@
 
-It is not possible to delete database files while `scdata` process is running.
+### OpenMode
 
-#### Linux
+The OpenMode option defines how to interpret the database path on connection. It can have the following string values:
 
-It is possible to delete database files while `scdata` is running, which may lead to a trap.
+Value               | Description
+:------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`Open`              | Opens the database directory from the path specified in the Database option, if it exist. If the directory does not exist, or if there is no Starcounter database located there, an exception is thrown.
+`CreateIfNotExists` | Opens the database directory from the path specified in the Database option, if it exist. If the directory does not exist, or if there is no Starcounter database located there, it will create the necessary directories and database files.
 
-Consider the following scenario:
+### StartMode
 
-- A database is created at `/home/database/TestDatabase`.
-- An `scdata` process is started for the database.
-- The database files are deleted.
-- New database is created in exactly the same location.
-- An app tries to connect to the new database.
-- Starcounter detects, that there is a running `scdata` process associated with this database path.
-- The app connects to the existing `scdata` process, which still operates on the deleted files.
+The StartMode option defines how to start and/or connect to the database process once the database files have been located and/or created.
 
-### The `DatabaseOpenMode` enumeration values
+Value               | Description
+:------------------ | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`RequireStarted`    | Expects the database process to already be running and will connect to it if it is, or throw an exception if it is not.
+`StartIfNotRunning` | Connects to the database process if it is already running, or starts and connects to it if it is not.
+`StartExclusive`    | Expects no database process to be already running, and starts and connects to a new process if that is the case. If a database process is already running, an exception is thrown.
 
-* `Open` - Opens the database if it exist.
-* `CreateIfNotExists` - Opens the database if it exist, and creates one if it doesn't.
+### StopMode
 
-### The `DatabaseStartMode` enumeration values
+The StopMode option defines whether to stop the database process once the connection to it is closed, for example when an application is terminated.
```

Comment:
```suggestion
The `StopMode` option defines whether to stop the database process once the connection to it is closed, for example when an application is terminated.
```

Reply:
Currently we have an issue, that an unexpected termination of an application leaves `scdata` process running. Works fine if the app shuts down properly.

Reply:
We should regard that as a bug, right? In which case it doesn't really change the intended semantics for `StopMode`

Reply:
Yes, just telling you in case you would want to test it.